Groep 3 devbranch

RainbowNova opened the pull request.

RainbowNova committed:
Rewind

RainbowNova committed:
yes

yes

RainbowNova committed:
Header component

Please push your entire most recent branch to main, merge and pull from main.
- I turnt the header into header-component so we only have to change code in one place.

RainbowNova commented:
Contains header component

JasperSmitz reviewed (approved):
I was with @RainbowNova when this was written. Works, approved.

YvoSchutgens reviewed (approved):
Looks good

RainbowNova committed:
Turnt login button into component

Same thing as the header:
instead of having 100 lines of code on every html page where the user should be able to login, it's now just 1 line of code which refers to a separate file.

RainbowNova committed:
Fixed last commit

RainbowNova committed:
Fixed components

Components now work (have tested it) on EVERY page.
